doc/examples/motion_correction.py

```diff
@@ -65,4 +65,4 @@
 #
 # .. footbibliography::
 #
-# .. include:: ../links_names.inc
+# .. include:: ../../links_names.inc
```

Comment:
to remove, this is automatically added by the generation script.  a forgotten case

Reply:
Removed in aa6db82.